### PR TITLE
Prototype for Profile page.

### PR DESCRIPTION
**Pull request by eselenic**
Some changes were from Week 0. Still working on designing page and will add more tests.

### CODE REVIEW

File: src/test/java/codeu/controller/ProfileServletTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	  @Test
+	  public void testDoGet_noUser() throws IOException, ServletException{
+		Mockito.when(mockSession.getAttribute("user")).thenReturn(null);
+	    profileServlet.doGet(mockRequest, mockResponse);
+	    
+	    //Checks if the user null, the redirected into login page
+	    Mockito.verify(mockResponse).sendRedirect("/login");
+
+	  }
+
```
Comment (Rafer45) [inline src/test/java/codeu/controller/ProfileServletTest.java]: Looks good! Could you add a test for when the user is not null?

EDIT: I just saw your message on adding more tests; understood!

File: src/main/java/codeu/controller/ProfileServlet.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+/** This servlet class is responsible the profile pages*/
+public class ProfileServlet extends HttpServlet{
+	
+	  /** From my perspective this variable is used to keep track of who is looking at
```
Comment (Rafer45) [inline src/main/java/codeu/controller/ProfileServlet.java]: The indentation is a little off from the one in the standard -- instead of tabs, there should be two spaces.

**Review by Rafer45 — CHANGES_REQUESTED** [review]

File: src/main/java/codeu/controller/ProfileServlet.java (hunk 2 above)
Comment (dgalbraith33) [reply src/main/java/codeu/controller/ProfileServlet.java]: Ditto on this. I'll try to get us set up with a linter soon that will catch formatting issues automatically. :smile:

Comment (jcolladokuri) [reply src/main/java/codeu/controller/ProfileServlet.java]: https://www.10bestdesign.com/dirtymarkup/js/

This websites has some cool formatting features for a couple languages

File: src/main/webapp/WEB-INF/view/profile.jsp [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<title>UserProfile</title>
+</head>
+<body>
+
+</body>
+</html>
```
Comment (dgalbraith33) [inline src/main/webapp/WEB-INF/view/profile.jsp]: nit: Add a newline at the end of the file

**Review by dgalbraith33 — CHANGES_REQUESTED** [review]

**Comment by Rafer45** [comment]
Looks good to me!

**Review by Rafer45 — DISMISSED** [review]

**Comment by Rafer45** [comment]
Looks good to me!

**Review by dgalbraith33 — DISMISSED** [review]
LGTM

**Review by dgalbraith33 — APPROVED** [review]
LGTM
